Explorer/Assets/Scripts/Global/Dynamic/PortableExperiences/ECSPortableExperiencesController.cs

```diff
@@ -26,27 +25,53 @@ public class ECSPortableExperiencesController : IPortableExperiencesController
         private readonly IWeb3IdentityCache web3IdentityCache;
         private readonly IWebRequestController webRequestController;
         private readonly IScenesCache scenesCache;
-        private readonly ObjectProxy<World> globalWorldProxy;
-
+        private readonly World globalWorld;
         private readonly List<IPortableExperiencesController.SpawnResponse> spawnResponsesList = new ();
         private readonly ServerAbout serverAbout = new ();
+        private readonly FeatureFlagsCache featureFlagsCache;
         public Dictionary<ENS, Entity> PortableExperienceEntities { get; } = new ();
-        private World world => globalWorldProxy.Object;
+
 
         public ECSPortableExperiencesController(
-            ObjectProxy<World> world,
+            World globalWorld,
             IWeb3IdentityCache web3IdentityCache,
             IWebRequestController webRequestController,
-            IScenesCache scenesCache)
+            IScenesCache scenesCache,
+            FeatureFlagsCache featureFlagsCache)
         {
-            globalWorldProxy = world;
+            this.globalWorld = globalWorld;
             this.web3IdentityCache = web3IdentityCache;
             this.webRequestController = webRequestController;
             this.scenesCache = scenesCache;
+            this.featureFlagsCache = featureFlagsCache;
+        }
+
+        public async UniTask<bool> CreatePortableExperienceByEnsWithErrorHandlingAsync(ENS ens, CancellationToken ct, bool isGlobalPortableExperience, bool force)
+        {
+            //I added this method to be able to handle errors that normally would be sent to JS to be dealt with, but when we call it from anywhere else (either initialization or chat commands),
+            // we are usually not prepared to handle them.
+            try {
+                    await CreatePortableExperienceByEnsAsync(ens, ct, isGlobalPortableExperience, force);
```

Comment:
There is a handy shortcut to do the same: take a look at `SuppressAnyExceptionWithFallback`, it contains special handling for OperationCancelledException as well to not spam with it

Reply:
Ohhh niiice